wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

```diff
@@ -44,22 +44,25 @@ The **GB Cup 2022 Autumn** (***GBC 2022 Autumn***) is a 3v3 captain draft-based
 | *17-21st place* | ¥60 CNY |
 | *22-24th place* | ¥30 CNY |
 
+![](img/badge_official.png "GBC 2022 Autumn winner profile badge") ![](img/badge_1st.png "GBC 2022 Autumn 1st KanonBot profile badge") ![](img/badge_2nd.png "GBC 2022 Autumn 2nd KanonBot profile badge") ![](img/badge_3rd.png "GBC 2022 Autumn 3rd KanonBot profile badge")
```

Comment:
why is there a winner badge *and* a first-place badge? what is "KanonBot"?

Reply:
winner badge is images used in osu! website.  

`KanonBot` is a third-party chat bot for osu!, it also have it own badge system, player could display those badge on their profile like osu! website.  
like this:   
![259UN6 {}7C@%XS_E8E(` P](https://user-images.githubusercontent.com/20500319/215259549-ecedea41-2eb0-49cd-8a25-19e3a4e0da40.PNG)